### PR TITLE
Get Travis CI automated build working

### PR DESCRIPTION
**Pull request by hiddentao**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+  depth: 1
+node_js: lts/*
+cache: yarn
 install:
-  - npm install -g truffle
-  - npm install -g ethereumjs-testrpc
-  - npm install
-env:
-  - TASK=test
-  - TASK=lint
```
Comment (makoto) [inline .travis.yml:12]: Is lint setup elsewhere or did you get rid of it?  This is not for eslint but for solium which runs linting against smart contract

**Comment by makoto** [comment]
after script to run coverall seems failing
screen shot 2018-08-29 at 07 32 22

File: .travis.yml (hunk 1 above)
Comment (hiddentao) [reply .travis.yml:12]: Fixed.

**Comment by hiddentao** [comment]
@makoto Please review again when you have time

**Comment by makoto** [comment]
@hiddentao are you setting up coverall on this PR or separate PR? If the latter good to go. If the former, the post deploy job is failing

screen shot 2018-08-29 at 14 46 58

**Review by makoto — APPROVED** [review]
Assuming coverall is setup as a separate PR

**Pull request merged**